_posts/2021-01-06-introducing-artsy-engineering-radio.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: epic
+title: "Introducing Artsy Engineering Radio"
+date: 2021-01-06
+categories: [podcast, community, learning]
+author: [steve-hicks, jonallured]
+---
+
+Available now [on iTunes][itunes] and hopefully your preferred podcast provider 🤞 is Artsy Engineering Radio!
+We'll release new episodes every other week. Our target episode length is 30 minutes. Some episodes will be
+technical and others less so. Because [we work in the open][open-source-by-default] it's easy for us to talk about
+our work in public.
+
+Solving problems in the world of software engineering can mean a lot of different things, and this podcast will
+explore what that looks like at Artsy. If you've followed our blog, you can expect a podcast that sounds like it.
+There are a ton of amazing engineers here at Artsy and we're excited for you to hear their voices and stories. Our
+hope is that this podcast will reduce the friction and get more contributions shared with the world.
+
+This isn't the first attempt at an Artsy engineering podcast! Our engineering team has attempted multiple times to
+start one but they fizzled. In fact, when [Jon][jon] went to create new Slack and Notion hubs, he found that both
+already existed. We're beating the fizzle this time, though. We've got momentum and enthusiasm on our side —
+several episodes are already recorded, a couple are already published. Instead of tinkering with tooling we're
+focusing on making episodes, even if they aren't perfect. Like the software we build, the podcast will get better
+over time as we iterate.
```

Comment:
Might be worth linking to the 2017 PR with an earlier attempt? https://github.com/artsy/artsy.github.io/pull/368